.github/workflows/run_tests.yml

```diff
@@ -21,7 +21,9 @@ jobs:
       fail-fast: false
       max-parallel: 5
       matrix:
-        python-version: [3.6, 3.7, 3.8] # , pypy3]
+        python-version: [3.6, 3.7, 3.8, pypy3]
```

Comment:
3.9 is now available. Probably more important to test different pandoc versions than python versions, although testing the python extremes (3.6 and 3.9) is definitely helpful

Reply:
Unfortunately this is a Python project. It seems that the standard is to test against all supported Python versions (as declared in setup.py.) Pandoc versions on the other hand, can be tested sparingly from experience, because the API is stable. Often you only need to test against "breaking boundaries", such as 1.17(?), 1.x, 2.0.x, 2.9.x, 2.11, latest. In my repos I sometimes toggle all pandoc versions once in a long while to make sure the gaps has no problem.

Reply:
also see commit 201815b and the comment above about pandoc versions supported.

when the PR is created Python 3.9 is not released yet.